src/entities/Product.java

```diff
@@ -0,0 +1,45 @@
+package entities;
+
+import enums.MeasurementUnit;
+
+public class Product extends BaseEntity<Long>{
+    private String name;
+    private Long buyingPrice;
+    private Long sellingPrice;
+    private MeasurementUnit measurementUnit;
+
```

Comment:
Did you mean MeasureUnit?

Reply:
would you like me to change the variable name to measureUnit? or the Enum class?

Reply:
Yeah makes more sense `MeasureUnit`

Reply:
solved